_episodes/06-plotting.md

```diff
@@ -0,0 +1,341 @@
+---
+title: "Plotting"
+teaching: 20
+exercises: 10
+questions:
+- "How can I plot some data?"
+- "How can I save my plot for publishing?"
+objectives:
+- "Import a library into a program using `import libraryname`."
```

Comment:
I am importing random in the previous lesson (no explanation given though, I thought I would just say it out loud)

Reply:
Ok. If you don't mind, and since your episode is about functions, can you insist a little bit of the fact that libraries provide us with functions already written (by other people) that we can readily use (call)?

And, when I 'introduce' how to import a library, I'll echo your episode (it won't hurt). Thanks!

Reply:
Hi @emilienschultz,

The above thread addresses the issue you opened at #15 :slightly_smiling_face: